website/docs/r/container_group.html.markdown

```diff
@@ -86,6 +111,22 @@ The `container` block supports:
 
 * `command` - (Optional) A command line to be run on the container. Changing this forces a new resource to be created.
 
+* `volume` - (Optional) The definition of a volume mount for this container. Changing this forces a new resource to be created.

```

Comment:
> `The definition of a volume mount for this container`

could we suffix this with "as documented in the `volume` block below"?

Reply:
Done.